README.md

```diff
@@ -4,20 +4,91 @@ This project is a ActiveMQ Artemis Self Provisioning Plugin to the Administrator
 
 ## Local development
 
+To be able to run the local development environment you need to:
+* have access to an OpenShift cluster
+* have the operator installed on the cluster
+* have the plugin running
+* have the console running
+
+### Setting up an OpenShift cluster
+
+In order to run the project you need to have access to an OpenShift cluster.
+You can either request one to the `Cluster bot` on slack, or install one on your
+local development environment.
+
+#### Remote cluster
```

Comment:
Is this available to community, or only for redhat people?

Reply:
That's a good question, I think it may only be accessible for redhat only, in that case what should we do about this ?

Reply:
we should omit it.

Reply:
I'll remove it. I'm wondering, is there some place I can describe the procedure with the Cluster bot? It could be helpful for new internal contributors also